opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -236,6 +236,7 @@ def _init_metrics(
 def _init_logging(
     exporters: Dict[str, Type[LogExporter]],
     resource: Resource = None,
+    setup_logging_handler: bool = True,
```

Comment:
This is True to keep compatibility with old callers. I know it's a private function but still.